tests/test_fixtures.py

```diff
@@ -0,0 +1,18 @@
+# third-party imports
+from django.conf import settings
+import pytest
+
+
+def test_request_page(request_page):
+    response = request_page("/dasmon/", settings.GENERAL_USER_USERNAME, settings.GENERAL_USER_PASSWORD)
+    assert response.ok
+
+
+def test_HTTPText(HTTPText):
+    text = HTTPText("on and on")
+    assert "on" in text
+    assert "ona" in text
```

Comment:
where does `ona` come from?

Reply:
[Here's the explanation](https://github.com/neutrons/data_workflow/blob/double_column/tests/conftest.py#L53):
```python
        def __contains__(self, substring: str):
            r"""check HTML string contains a substring, but whitespaces are not considered
            :param substring: substring to match
            """
```